pkg/rebuild/debian/infer.go

```diff
@@ -79,20 +82,13 @@ func (Rebuilder) InferStrategy(ctx context.Context, t rebuild.Target, mux rebuil
 						deps[i] = strings.TrimSpace(strings.Split(dep, " ")[0])
 					}
 				}
-				dependencies = append(dependencies, deps...)
+				p.Requirements = append(p.Requirements, deps...)
 			}
 		}
 	}
-	if (orig.URL == "" || debianTar.URL == "") && (native.URL == "") {
-		return nil, errors.New("Failed to find source files in the .dsc file")
+	if (p.Orig.URL == "" || p.Debian.URL == "") && (p.Native.URL == "") {
+		log.Printf("Partially constructed strategy: %v", p)
```

Comment:
are you wedded to logging this? i think i'd lean away from logging here but I am open to it

Reply:
To elaborate, the cost to reconstruct this data seems quite low while the range of cases you're trying to account for is unclear. Could you instead log unknown suffixes?